### ISSUE TITLE
Regeneration Passive skill in Trolls makes them Immortal during Swimming

### ISSUE THREAD

**Issue by Helieus** [LightsHope/server#878]
Regeneration : Trolls Regenerate 10% of their health , in combat and out of combat.

Breathing Damage : For A level 17 it would be between 12-48 points

My Character : 558 Health Points . 

The Regeneration : 558*0.1  =  55.1 <- This is the Regenerated health per SECOND 

If I lose 48 Health Points I will regenerate 55 therefore I am Immortal.

**Comment by Seberats** [comment]
No. Troll regeneration is "increased" by 10%. Its not 10% of max health, every second.

**Comment by Cabro** [comment]
Why is it regenerating 10% of total HP?! It should be 10% of health regeneration, not total HP dafuq...

**Comment by Omarthegreatone** [comment]
Make trolls great again!
